### DISCUSSION TITLE
There is an issue with the new version's support for Jansi colors, which cannot be displayed correctly

### DISCUSSION THREAD

**Discussion by mingyang66** [qos-ch/logback#702] [Q&A]
There is an issue with the new version's support for Jansi colors, which cannot be displayed correctly

**Comment by mingyang66** [comment]
Logback Version is 1.4.11, and jansi version is 2.4.0

**Comment by ceki** [comment]
@mingyang66 Can you provide details about your OS? A test case would be nice.
